### DISCUSSION TITLE
Duda de entrega HTML

### DISCUSSION THREAD

**Discussion by OscarIslasH** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#274] [Q&A]
Tengo una ultima duda al entregar la kata de html , en lo de lenguaje solo sale html pero use tambien css ¿Es normal o afecta a algo?

aqui esta mi repositorio por cualquier cosa: https://github.com/OscarIslasH/Pasteleria

![Captura de pantalla 2022-03-06 204251](https://user-images.githubusercontent.com/99118114/156959093-4c33cc81-eb68-4be7-8175-07cf16f29c9e.png)

**Comment by doguedogue** [comment]
Es normal NTP, es algo que contabiliza GitHub como de lo que hay más. Con que este tu archivo arriba no hay problema

**Reply by OscarIslasH** [reply]
Gracias !

**Comment by JuanSalvador73** [comment]
Entré a ver tus archivos y dentro de ese folder están los que mencionas. No tienes ningún problema.

**Comment by hackit92** [comment]
Todo parece estar arriba, sin embargo te recomiendo que estandarices la organización de tus archivos, generalmente tus archivos de "páginas" estarán en la raíz para evitar navegación hacia arriba y luego hacia abajo, es decir... assets, js, css en carpetas, los .html .php en la raiz
